.helix/languages.toml

```diff
@@ -0,0 +1,5 @@
+[language-server.rust-analyzer.config.cargo]
+features = "all"
+
+[language-server.rust-analyzer.config.server.extraEnv]
+RUSTUP_TOOLCHAIN = "stable"
```

Comment:
This is very similar to the `.vscode/settings.json` file that includes extra features for the IDE's `rust-analyzer` instance.